SeqGAN/obama/obama_helper.py

```diff
@@ -0,0 +1,99 @@
+
+import json
+import random
+
+# Helper functions for converting between Haiku to DataLoader form
```

Comment:
Haiku?

Reply:
There is a lot of code reuse between the obama_helper.py and haiku_helper.py. I would take both directories out of SeqGan and merge the two *_helper.py filles into a top_level dataprocessing.py file that reuses as many functions as possible. I'd also put the data (haiku.train.txt, haiku.valid.txt, obama.train.txt, obama.valid.txt) into the topmost data directory.

Reply:
I fixed the data processing files issue by adding a dataloader_helper.py file and removing the old helper files. I thought though that keeping the data files (haiku.train.txt, haiku.valid.txt, obama.train.txt, obama.valid.txt) in the subdirectories makes sense because it makes each application a separate module. What do you think? Does that work with the way I restructured it? @mhrmm